SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -298,6 +298,10 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Runtime", meta = (DisplayName = "Runtime versions"))
 	FRuntimeVariantVersion StandardRuntimeVersion;
 
+	/** Enable to terminate the runtime after exiting from PIE. This reduces the shut down time of the runtime process. */
```

Comment:
Does this setting not increase the time it takes to shut down?

Reply:
To shut down pie technically yes. It does not affect dev time though ie you don't have to wait 5 seconds before you can interact with the editor. You can hit play again as soon as you stop pie. You will have to wait for the runtime to shut down cleanly and then open again though before you can actually play.

Reply:
Interesting, I feel like the description is describing the force termination rather than the graceful shutdown (where the force termination would reduce the shutdown time)

Reply:
Yeah I agree with Danny here, the description is saying "terminate the runtime" but the setting is "ShutdownRuntimeGracefully", seems the description just needs bringing in-line

Reply:
Indeed. Enable -> Disable